$OpenBSD: patch-core_CMakeLists_txt,v 1.1 2019/06/16 15:14:56 rsadowski Exp $

Generate Info.plist only one time and create a dependency. Fixes a problem in
combination with Ninja v1.9.0 because of duplicate rules for generating
Info.plist.

Index: core/CMakeLists.txt
--- core/CMakeLists.txt.orig
+++ core/CMakeLists.txt
@@ -640,6 +640,8 @@ message(STATUS "")
 
 if(DIGIKAM_CAN_BE_COMPILED)
 
+    configure_file(${CMAKE_CURRENT_SOURCE_DIR}/cmake/templates/Info.plist.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/Info.plist)
+    add_custom_target(generate_info_header DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/Info.plist)
     # ==========================================================================
     # Create git version header
 
